.gitignore

```diff
@@ -14,3 +14,4 @@ doc/api/generated
 *.egg-info
 MANIFEST
 .coverage.*
+.vscode/settings.json
```

Comment:
```suggestion
```

@luke-gregor you might want to set this in your global `~/.gitignore_global` instead.